bliss/api.py

```diff
@@ -16,6 +14,7 @@
 from bliss.predict import predict_sdss as _predict_sdss
 from bliss.surveys import sdss_download
 from bliss.train import train as _train
+from bliss.utils.download_utils import download_git_lfs_file
```

Comment:
Probably better to avoid creating a "utils" module, and instead to keep the relevant code near where it's being used (i.e., within the surveys module)

Reply:
See https://www.yanglinzhao.com/posts/utils-antipattern/